server/src/test/java/org/apache/iotdb/db/mpp/sql/plan/DistributionPlannerTest.java

```diff
@@ -77,6 +84,47 @@ public void TestRewriteSourceNode() throws IllegalPathException {
 
     System.out.println(PlanNodeUtil.nodeToString(newRoot));
     assertEquals(newRoot.getChildren().get(0).getChildren().size(), 3);
+
```

Comment:
Suggest to separate the test into two tests.

Reply:
fixed